go/vt/vtgate/executor.go

```diff
@@ -1004,6 +1025,10 @@ func (e *Executor) handleOther(ctx context.Context, safeSession *SafeSession, sq
 	}
 	execStart := time.Now()
 	result, err := e.destinationExec(ctx, safeSession, sql, bindVars, dest, destKeyspace, destTabletType, logStats)
+
+	queriesProcessed.Add("Other", 1)
+	queriesRouted.Add("", int64(logStats.ShardQueries))
```

Comment:
Hm. Why "" here as a plan type and not just "Other"?

Reply:
Because I didn't copy/paste correctly.
Thanks, nice catch. Fixed.